src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,22 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.3.0
+
+### New Feature
+
+- Git Catalog to provide predefined development environments to every developer
+- Add Git Catalog tab to Okteto UI deploy
+- Remove CLI tab from Okteto UI deploy
+
+### Bug Fix
+
+- Use case insensitive check for GitHub ID
+- GitHub redeploy modal was allowing `repo` field to be modified if opened fast enough
+- Support for "resources" for the okteto daemonset
```

Comment:
This should be in the Feature lists. Can we also add a link to the configuration doc for this?

```suggestion
- You can now define  "resources" for the okteto daemonset component. Use this to ensure that Kubernetes will reserve resources for the `daemonset`.
```